packages/turf-dissolve/index.js

```diff
@@ -0,0 +1,161 @@
+var turfUnion = require('@turf/union');
+var turfOverlaps = require('turf-overlaps');
+var turfbbox = require('@turf/bbox');
+var Rbush = require('rbush');
+var gju = require('geojson-utils');
+var getClosest = require('get-closest');
+
+/**
+ * Dissolves a FeatureCollection of polygons based on a property. Note that multipart features within the collection are not supported
+ *
+ * @name dissolve
+ * @param {FeatureCollection<Polygon>} featureCollection input feature collection to be dissolved
+ * @param {string} propertyName property name on which to dissolve features
```

Comment:
That works for me, I just know that QGIS & ArcGIS can dissolve using multiple attributes, just something to keep in mind in the future.